src/select.rs

```diff
@@ -20,11 +23,15 @@ pub struct Select<T> {
     items: Vec<RadioButton<T>>,
     cursor: usize,
     initial_value: Option<T>,
+    enable_filter_mode: bool,
+    switch_filter_mode: bool,
+    filter: StringCursor,
+    filtered_items: Vec<RadioButton<T>>,
```

Comment:
The problem is that I use this filtered item struct to send the right item when pressing enter 
For example : 
items unfiltered : 
A
B
C
D

items filtered : 
A
D

Do you have any idea on how I could jump the cursor from A to D in case like this ?

Reply:
@antoinePutmans I _think_ that having the _max-list-display-length_ feature that I mentioned in the issue might be of help to you here -- if we say in this example that that was set to `4` for this list (A, B, C, D), then to avoid the terminal jumping around (which would be mostly noticeable when the `select` is at the bottom of a scrolled terminal) you would most likely want to keep all four of those lines and simply clear the unused ones. Then you still have your 4 rows, know your "row positions" vs your item positions and can use `console` to navigate up/down on render to the currently selected item.

Hope that made sense? :smile:

Reply:
@antoinePutmans Oh, I got the point,  you're right.
@cylewitruk he doesn't mean a terminal rendering issue, he means the internal logic of the cursor behavior (up/down).

Reply:
> @antoinePutmans Oh, I got the point,  you're right.
> @cylewitruk he doesn't mean a terminal rendering issue, he means the internal logic of the cursor behavior (up/down).

Aha, okay :)

Reply:
@cylewitruk I will try to implement the max length feature in an other PR 👍